cellfinder_napari/detect.py

```diff
@@ -1,14 +1,22 @@
-import napari
+from math import ceil
```

Comment:
I'm curious, I've never found a need to use `math` over `numpy` unless trying to cut down on dependencies. Is that the idea here?

Reply:
I've always just used in built functions over imported libraries. No real reason other than simplicity.